File: app/src/main/java/com/surya/shopcart/cart/EmptyCartActivity.java
package com.surya.shopcart.cart;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.surya.shopcart.ProductHomePageActivity;
import com.surya.shopcart.R;
import com.surya.shopcart.utils.Utils;

public class EmptyCartActivity extends AppCompatActivity {


    public void openCartPage(MenuItem item) {
        Utils.handleMenuCLick(this, item);
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_empty_cart);

        // Home navigation icon
        Utils.addHomeIconNavigation(this, findViewById(R.id.topAppBar));

        Button homePage = findViewById(R.id.homePage);
        homePage.setOnClickListener(view -> {
            Intent homePageIntent = new Intent(this, ProductHomePageActivity.class);
            startActivity(homePageIntent);
            finish();
        });
    }
}

File: app/src/main/java/com/surya/shopngo/cart/EmptyCartActivity.kt
package com.surya.shopngo.cart

import android.content.Intent
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.surya.shopngo.ProductHomePageActivity
import com.surya.shopngo.R
import com.surya.shopngo.utils.Utils

class EmptyCartActivity : AppCompatActivity() {
    fun openCartPage(item: MenuItem) {
        Utils.handleMenuCLick(this, item)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_empty_cart)

        // Home navigation icon
        Utils.addHomeIconNavigation(this, findViewById(R.id.topAppBar))
        val homePage = findViewById<Button>(R.id.homePage)
        homePage.setOnClickListener { view: View? ->
            val homePageIntent = Intent(this, ProductHomePageActivity::class.java)
            startActivity(homePageIntent)
            finish()
        }
    }
}